File: 이제준/src/WEEK2.java
import java.util.Arrays;

class Solution {
    public int solution(int[] people, int limit) {
        Arrays.sort(people);

        int min = 0;
        int answer = 0;

        for(int i = people.length -1; min <= i; i--){
            if(people[i] + people[min] <= limit){
                min++;
            }
            answer++;
        }
        return answer;
    }
}

File: JeongMeoYoon/WEEK2.js
function solution(citations) {
  let answer = 0;

  citations.sort((x, y) => x - y); // 일단 정렬을 진행한다.
  citations.forEach((value, index) => {
    const n = Math.min(citations.length - index, value);
    answer = Math.max(answer, n);
  });
  return answer;
}
